.github/workflows/update-no-op-plugin.yml

```
@@ -0,0 +1,38 @@
+name: Remove OP Plugin
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  remove-op-plugin:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

We need to use the latest version to avoid warnings:

```suggestion
      - name: Check out repository
        uses: actions/checkout@v4
```

---

thanks